ertviz/models/plot_model.py

```diff
@@ -134,10 +134,17 @@ def repr(self):
         colors = []
         data = []
         names = []
+        realization_num = []
```

Comment:
Nice, was wondering if you could change the variable name to plural, eg. realization_nums, for the sake of clarity that we talk many.

Reply:
Sure!